docs/trex_security.md

```diff
@@ -30,7 +30,7 @@ When you use HTTPS, all HTTP data is encrypted prior to transmission by the Tran
 If you plan to make your extension available to others, using HTTPS
  assures your customers that their data is safe and that they are connecting to a trusted extension. Because the extension is using HTTPS, Tableau is also able to verify the identity of the server that hosts the extension, which prevents various malicious man-in-the-middle attacks that could occur if the extension were to use HTTP alone.  
 
-> Note: For development or internal use, you can run your extension on your local computer as `localhost` and you can use the HTTP protocol. See [Use HTTP and localhost for development or internal use](#use-http-and-localhost-for-development-or-internal-use).
+| Note: For development or internal use, you can run your extension on your local computer as `localhost` and you can use the HTTP protocol. See [Use HTTP and localhost for development or internal use](#use-http-and-localhost-for-development-or-internal-use).
```

Comment:
This should be `>`.  The Markdown for a block quote is a `>`, which is what we are using here. And it gets rendered as a vertical line. 
It's not the best. It would look better if we used HTML "note" for it, but a that's not great either.